integrationtests/server-integration/wildfly-modules/pom.xml

```diff
@@ -9,7 +9,7 @@
       <relativePath>../pom.xml</relativePath>
    </parent>
 
-   <artifactId>infinispan-wildfly-module-integrationtests</artifactId>
+   <artifactId>infinispan-server-integrationtests-wildfly-modules</artifactId>
```

Comment:
I think this name is better :D

Reply:
-1

Reply:
IMO, it sounds like you are testing the WF modules in infinispan server... which makes no sense.